inc/TRestGeant4ParticleSourceDecay0.h

```diff
@@ -0,0 +1,38 @@
+#ifndef TRestParticleSourceDecay0_Class
+#define TRestParticleSourceDecay0_Class
+
+#include <bxdecay0/decay0_generator.h>
+#include <bxdecay0/event.h>
+#include <bxdecay0/std_random.h>
+
+#include <iostream>
+
+#include "TObject.h"
+#include "TRestMetadata.h"
+#include "TRestGeant4ParticleSource.h"
+
+using namespace std;
```

Comment:
We should avoid `using namespace std;` in the header

Reply:
That's a good point